apps/air-discount-scheme/backend/src/environments/environment.ts

```diff
@@ -54,6 +63,18 @@ const prodConfig = {
     url: process.env.NATIONAL_REGISTRY_URL,
     username: process.env.NATIONAL_REGISTRY_USERNAME,
     password: process.env.NATIONAL_REGISTRY_PASSWORD,
+    authMiddlewareOptions: {
+      forwardUserInfo: false,
+      tokenExchangeOptions: {
+        issuer: isProd
+          ? getStaticEnv('SI_PUBLIC_IDENTITY_SERVER_ISSUER_DOMAIN')
+          : '',
```

Comment:
Same issue as above

Reply:
Without it there will be errors when running it locally.